packages/synapse-react-client/src/components/ChallengeTeamWizard/JoinRequestForm.tsx

```diff
@@ -1,25 +1,43 @@
 import React, { ChangeEvent, useState } from 'react'
-import { Box, TextField, Typography } from '@mui/material'
-import { Team } from '@sage-bionetworks/synapse-types'
+import { Alert, Box, TextField, Typography } from '@mui/material'
+import { useGetTeam } from '../../synapse-queries'
+import { SynapseSpinner } from '../LoadingScreen/LoadingScreen'
 
 type JoinRequestFormProps = {
-  team: Team | undefined
+  teamId: string | undefined
   joinMessageChange: (msg: string) => void
 }
 
 export const JoinRequestForm = ({
-  team,
+  teamId,
   joinMessageChange,
 }: JoinRequestFormProps) => {
```

Comment:
Update a few components to use a teamId instead of the full team object. The component will fetch the team itself (and probably just hit the cache)